include/system/readline.h

```diff
@@ -171,6 +171,16 @@ FAR const struct extmatch_vtable_s *
  *
  ****************************************************************************/
 
+ssize_t readline_fd(FAR char *buf, int buflen, int infd, int outfd);
+
+/****************************************************************************
+ * Name: readline
+ *
+ *   readline() is same as readline_fd() but accept the file stream instead
+ *   of file handle.
```

Comment:
```suggestion
 *   readline() is similar to readline_fd() but accepts a file stream instead
 *   of a file descriptor.
```
nit:  Regarding description.

Reply:
Done.